### PR TITLE
Remove patch from xliff-tasks build

### LINKED ISSUE #180: Remove patches from xliff-tasks
Remove all from https://github.com/dotnet/source-build/tree/dev/release/2.0/patches/xliff-tasks

### PR DESCRIPTION
**Pull request by nguerrera**
Fix #180

The patch was a convoluted way to only build and pack the main project and not the tests. . I changed the targets to just build and test the one project directly, same as restore just above.

This will surely change with repo API, but in the meantime, it is trivial not to have any xliff-tasks patches this way.

### CODE REVIEW

File: targets/xliff-tasks.targets [hunk 1]
```diff
@@ -3,9 +3,14 @@
...
     <Message Importance="High" Text="Building '$(RepositoryName)'" />
 
-    <Exec Command="$(DotNetCommand) restore $(ProjectDirectory)src\XliffTasks\XliffTasks.csproj" />
+    <!-- Build only the tasks project, not the tests. -->
+    <PropertyGroup>
+      <_XliffTasksProject>$(ProjectDirectory)src\XliffTasks\XliffTasks.csproj</_XliffTasksProject>
+    </PropertyGroup>
 
-    <Exec Command="$(DotNetCommand) build $(ProjectDirectory)build\build.proj /t:Pack /v:normal /flp:Verbosity=Diag;LogFile=$(ProjectDirectory)msbuild.log"
+    <Exec Command="$(DotNetCommand) restore $(_XliffTasksProject)" />
```
Comment (eerhardt) [inline targets/xliff-tasks.targets]: Isn't `dotnet restore` implicit now?

Comment (nguerrera) [reply targets/xliff-tasks.targets]: It is. I could collapse this to just pack.

Comment (nguerrera) [reply targets/xliff-tasks.targets]: Done.

**Comment by nguerrera** [comment]
> we will need to revisit this when we implement the "repo API"

Yep, I said the same thing in the PR description. :)

**Comment by nguerrera** [comment]
@eerhardt Good to merge?

**Comment by eerhardt** [comment]
Yep.  @crummel is working on fixing the Tizen build.

**Pull request merged**